### DISCUSSION TITLE
Running Connect-AzAccount interactively does not work

### DISCUSSION THREAD

**Discussion by chaosdinosaur** [dotnet/interactive#3267] [Q&A]
I have a project where -UseDeviceAuthentication does not work for some Non-Prod subscriptions when trying to connect to Azure. 

When I try using Connect-AzAccount without this flag, I get:

```
WARNING: Interactive authentication is not supported in this session, please run cmdlet 'Connect-AzAccount -UseDeviceAuthentication'.
```

Is there a work around for this or can this be something that is fixed?

I'm running my repo in Codespaces and using a PowerShell cell.

**Comment by jonsequitur** [comment]
@daxian-dbw You probably have more context for this question.
